doc/main.html

```diff
@@ -436,6 +438,28 @@ <h4>Foreword section</h4>
 <pre>&lt;section id=&quot;sec-foreword&quot;&gt;
   &lt;p&gt;This is the additional information relevant to the document.&lt;/p&gt;
 &lt;/section&gt;
+</pre>
+        </div>
+
+        <p>When <code>pubType</code> is set to <code>RDD</code>, the <a>Foreword</a> section shall be present and there there shall be a single <code>dl</code>, after any author-supplied prose if supplied, which contains contact information about the proponent(s) of the RDD document. 
+
+        <p>The <code>id</code> attribute shall be present on the <code>dl</code> element and equal to
+        <code>rdd-proponent</code>. 
+
+        <p>If present, the <code>dl</code> element, as described in <a href="#sec-dl-element"></a>, shall use a single <code>dt</code> element for each company name designated as a Proponent of the document, and multiple lines of <code>dd</code> as needed for contact information for each company listed. The contact infomation should at least include an address and email, reach out to SMPTE HO for further deatails. </p> 
```

Comment:
Isn't a single <dd>, with potentially multiple lines? Each <dd> is supposed to be a synonym of the <dt>, not a separate line

Reply:
... in fact, I am not convinced `<dl>` is the right approach if there are multiple proponents... can you provide an example of how this would work?

Reply:
```
<dl id="element-proponent">
 
      <dt>Company name</dt>
        <dd>Contact name</dd>
        <dd>Contact email</dd>
        <dd>Company Address</dd>
        <dd>Company website</dd>
     
      <dt>Company 2 name</dt>
        <dd>Contact name</dd>
        <dd>Contact email</dd>
        <dd>Company 2 Address</dd>
        <dd>Company 2 website</dd>
   
</dl>
  ``` 
Nothing prevents the usage of multiple `dd` for each `dt`, such as https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#single_term_multiple_descriptions. This is frankly the least ugly way to present, besides trying to to the same with `ul` and then wrapping a style in CSS.

Reply:
I added the above example to the documentation as well.